protoc-gen-openapiv2/internal/genopenapi/template_test.go

```diff
@@ -2616,7 +2617,7 @@ func TestApplyTemplateRequestWithBodyQueryParameters(t *testing.T) {
 								PathTmpl: httprule.Template{
 									Version:  1,
 									OpCodes:  []int{0, 0},
-									Template: "/v1/{parent=publishers/*}/books",
+									Template: "/v1/publishers/{parent}/books",
```

Comment:
Why did you change this test?

Reply:
this is my mistake, ran a "replace all" operation but I only meant to change the result paths below, not this template

Reply:
OK! All good.